src/components/ReviewsContainer/ReviewsContainer.jsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react"
+
+import DropdownMenu from "./DropdownMenu"
+import ListOfReviews from "./ListOfReviews"
+
+import * as api from '../../api'
+
+const ReviewsContainer = () => {
+
+    const [reviews, setReviews] = useState()
+
+    useEffect(() => {
```

Comment:
we should have some loading state and render something for the user experience so they know data is loading if/when it is and same for error

Reply:
were also not using the reviews in this element, we're just passing it all down, therefor we could do this in the child